frameworks/hdfs/tools/docker-client/krb5.conf

```diff
@@ -0,0 +1,21 @@
+[libdefaults]
```

Comment:
In the tests that I did (including active directory), one can get away with the following minimal `krb5.conf`:
```
[libdefaults]
default_realm = MY.REALM

[realms]
  MY.REALM = {
    kdc = KDC_ADDRESS
  }
```
(see https://github.com/mesosphere/dcos-commons/blob/master/testing/security/kerberos.py#L33)

As already discussed, however, we could consider just dropping it.